docs/en/latest/getting-started.md

```diff
@@ -172,9 +170,7 @@ curl "http://127.0.0.1:9080/apisix/admin/upstreams/1" -H "X-API-KEY: edd1c9f0343
 
 We use `roundrobin` as the load balancing mechanism, and set `httpbin.org:80` as our upstream target (Upstream service) with an ID of `1`. For more information on the fields, see [Admin API](./admin-api.md).
 
-:::note Note
```

Comment:
The `:::note` has special meanings. CC @juzhiyuan

Reply:
![image](https://user-images.githubusercontent.com/2106987/126922842-44091ba6-cdff-49e0-b060-6de90a95e777.png)

Hi, this is a mark from Docusarus framework, which is used on our website :) e.g https://apisix.apache.org/docs/apisix/getting-started

Reply:
So now how to deal with it？Reading on Github will be affected😑

Reply:
Why not explore the docs from http://apisix.apache.org/docs/apisix/getting-started ?

Reply:
Docs under the Apache APISIX repo is the data source for APISIX Website.

Reply:
emmm, prefer to look on Github, the code and documentation are in one place... So `:::note` will cause a bad reading experience. Of course you can turn off this PR if you decide you don't need to change it.

Reply:
What about adding comments for them? So the new comer will understand it.

Reply:
How about adding some descriptions here ?https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contribution-guidelines-for-documentation

Reply:
Few people read the guide. It would be better to add a comment next to the mark.

Reply:
let's cc @liuxiran @yzeng25  What do you think?

Reply:
IMO we should keep it as it is right now. 

Usually, contents behind or under `Note:` are something important or needed to be emphasized.

The origin `Note:` is somewhat ambiguous (it could be note, tips, warnings or even fatal warnings, depends on how you modify or define them). Besides, plain text 'Note' is easy to be ignored or skipped. The purpose of using `:::note` and `:::` to quote a note is to make it more noticeable on docs website. 

Yes, this is not convenient for GitHub readers. But why don't you checkout APISIX document website? This alternative is better.

Reply:
We need to add a comment to clarify them so that I don't need to explain it again and again.

Reply:
OK.

Reply:
If so, there is a problem of uniformity of formatting in the document, and it feels like the usage of Note is not the same in many places.
https://github.com/apache/apisix/blob/6af1b69607bfb26bb6b61ec3b8307e2425e1f1c3/docs/en/latest/getting-started.md#L79

https://github.com/apache/apisix/blob/e3e018315311e0ca201f17579f9106eb88a834d1/docs/en/latest/plugin-develop.md#L90

https://github.com/apache/apisix/blob/baf843403461883c1334e63d15a6bb3622c31940/docs/en/latest/plugins/proxy-cache.md#L26

https://github.com/apache/apisix/blob/64923b3190384512a85f48c425b3ace0e5b32467/docs/en/latest/install-dependencies.md#L31

Reply:
Agree to keep it as it is right now. We can guide the user through the documentation, and actually make contributions without looking documentation is not a recommended way.

Reply:
APISIX's website serves Apache APISIX and other subprojects. Those markdown files are the source data for the website. If we admit it's a **product** just like Apache APISIX, then I would still prefer to keep the mark, so documentation readers would have a better experience than origin markdown files. I encourage people to use `http://apisix.apache.org/docs` instead of markdown files.

As for people ask why there has such a mark, what I can do is only adding that notice from Docusarus to CONTRIBUTING Guide, `Few people read the guide` doesn't mean this is not appropriate.

Reply:
True, but all 'notes', as you figured out in comments above, should transform to `:::note Note` gradually.

Reply:
I can understand, later I go to the official website to read the documentation. 😊

Reply:
@Applenice , thanks. 
If you find any potential typo in the future, please check out our documentation website `http://apisix.apache.org/docs` first. It is very likely to be a user experience enhancement on the website.

Reply:
I think the point you made might be better if it was noted prominently in the documentation. Some people may be used to reading it on Github, or they may be reading it offline after the project is downloaded, but after stating it more people will know that the reading experience on the official website is better than now, and they may avoid PR like this.

Reply:
Yep, that's what happened in PR https://github.com/apache/apisix/pull/4681/

Reply:
Thanks to everyone😊